src/client.cpp

```diff
@@ -1775,8 +1783,10 @@ void Client::afterContentReceived(IrrlichtDevice *device)
 	delete[] tu_args.text_base;
 
 	// Start mesh update thread after setting up content definitions
-	infostream<<"- Starting mesh update thread"<<std::endl;
-	m_mesh_update_thread.start();
+	infostream<<"- Starting mesh update thread pool"<<std::endl;
+	int thread_count = 1;
```

Comment:
@est31 Oddly there is no `Settings::getU32`(`NoEx`)`()`. Will keep `s32` until it gets implemented.